charts/__tests__/DataTable.test.tsx

```diff
@@ -124,4 +124,19 @@ describe(DataTable, () => {
             expect(notice.prop("targetYear")).toBe("2016")
         })
     })
+
+    describe("when you try to hide countries", () => {
```

Comment:
Nice!

Not sure if it would be easy, but for tests like these I like a before and after assertion.

Something like:

```typescript
expect(view.find("tbody tr")).toHaveLength(200)
// add filter
expect(view.find("tbody tr")).toHaveLength(187)
```

Also, will be nice someday when we can create simple charts on the fly like:

```typescript
const chart = new Chart({data: `country,pop,gdp
usa,123,123`, minPopulationFilter: 1e6})
...
```

Reply:
Ah, so that test actually does already exist up at:
 https://github.com/owid/owid-grapher/blob/2e728d79f510019fd05c5338ddbf9984aae1759a/charts/__tests__/DataTable.test.tsx#L41-L43. I can copy-paste it to test the transition.

Reply:
Oh nice, I didn't see that context.

Reply:
Copied it over to make it more explicit.